docs/jwst/flatfield/reference_files.rst

```diff
@@ -206,8 +206,20 @@ The DQ_DEF table contains the bit assignments used in the DQ array, and contains
 
 *IFU*
 ~~~~~
-The IFU reference files have EXP_TYPE=NRS_IFU, a BINTABLE
-extension labeled FAST_VARIATION, and a BINTABLE labeled DQ_DEF.
+The IFU reference files have EXP_TYPE=NRS_IFU.  There are 3 IMAGE extensions,
```

Comment:
The IFU fflat reference files have only one extension, a BINTABLE labeled FAST_VARIATION.